tdrs-backend/tdpservice/search_indexes/documents/document_base.py

```diff
@@ -1,16 +1,18 @@
 """Elasticsearch base document mappings."""
 
-from django_elasticsearch_dsl import fields
+from django_elasticsearch_dsl import fields, Document
 from tdpservice.data_files.models import DataFile
 
-class DocumentBase:
-    """Elastic search model mapping for a parsed SSP M1 data file."""
+
+class DocumentBase(Document):
+    """Elastic search base document to be overridden."""
 
     datafile = fields.ObjectField(properties={
-                      'pk': fields.IntegerField(),
+                      'id': fields.IntegerField(),
```

Comment:
only thing i would say here is that `id` is not always the `pk` and we've run into cases in the past where this assumption has broken down. everything works in this case, though